static/js/skeleton.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+
+	setTimeout(function() {
+		$('.skeleton-container').remove();
+		$('.skeleton-input').remove();
+		$('.skeleton').removeClass('skeleton');
+		$(".loaded").attr('style', 'display: block !important');
+	}, 500);
```

Comment:
Are we sure about the 500 ms As you mentioned, it might differ for other components in different pages.
As per this [answer](https://stackoverflow.com/questions/25831679/select2-initialization-callback#answer-25833086), we can get it working by simply removing the skeleton classes after the `.auiSelect2()`.

Reply:
Guess it depends if we want a blanket skellie or individual. This will work on the indiviual element but could leave us with half a skellie page?

Reply:
We can play around with that if needed when updating other pages. I don't follow what you mean: 'we can get it working by simply removing the skeleton classes after the .auiSelect2().' ??

Reply:
In that case, it's fine. 
It would be better to change this into a function and make `500` as default for a parameter, so that we can set different values if used in other components(optional).

Reply:
Can we remove skeleton classes on select2 load event?

Reply:
No, `select2-load` event is triggered only when the dropdown is opened.

Reply:
> It would be better to change this into a function and make 500 as default for a parameter, so that we can set different values if used in other components(optional).

We won't know that though until we implement it in other parts of the app. My inclination is to leave as is for and then, when someone picks up the work to switch from the spinner to skeleton, if we determine the 0.5 sec wait time doesn't work, refactor it. I don't want to refactor it now only to find out every location that uses it winds up passing the same value.